components/ADempiere/PanelInfo/Component/Accounting/index.vue

```diff
@@ -499,17 +540,43 @@ export default defineComponent({
       }
     }
 
-    if (isEmptyValue(accoutingSchemaId.value)) {
+    function exportAccounting() {
+      const fileName = listAccouting.value.find(list => list.KeyColumn === accoutingSchemas.value).DisplayColumn
+      console.log({ fileName })
+      exportFileFromJson({
+        header: headerAccounting.value.map(a => a.label),
+        data: formatJson(headerAccounting.value.map(a => a.columnName), tableData.value),
+        fileName,
+        bookType: 'xlsx'
+      })
+    }
+
+    function formatJson(filterVal, jsonData) {
+      console.log({ filterVal, jsonData })
```

Comment:
please remove this console

Reply:
Resolve in next Commit https://github.com/solop-develop/frontend-default-theme/pull/681/commits/8ea0f49be74a586ca79fd7687f096a278307574b